site/zenodo_rdm/legacy/serializers/schemas.py

```diff
@@ -63,6 +66,90 @@ def dump_resource_type(self, result, original, **kwargs):
                 result[f"{upload_type}_type"] = resource_type_id.split("-")[-1]
         return result
 
+    def _funder(self, funder):
+        """Serialize RDM funder into Zenodo legacy funder."""
+        legacy_funder = {"name": funder["name"]}
+
+        f_identifiers = {}
+        for scheme, identifier in funder.get("identifiers").items():
+            if scheme == "oaf":
```

Comment:
We might need to modify [VOCABULARIES_FUNDER_SCHEME](https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/config.py#L49). At the moment it does not accept oaf.